### PR TITLE
bump up version to 3.9.1

### PR DESCRIPTION
**Pull request by rashmi43**

### CODE REVIEW

File: deploy/olm-catalog/ibm-iam-operator/3.9.1/ibm-iam-operator.v3.9.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1991 @@
...
+                        - s390x
+              containers:
+              - command:
+                - ibm-iam-operator
+                env:
+                - name: ICP_PLATFORM_AUTH_IMAGE
+                  value: quay.io/opencloudio/icp-platform-auth:3.4.0
+                - name: ICP_IDENTITY_PROVIDER_IMAGE
+                  value: quay.io/opencloudio/icp-identity-provider:3.4.0
+                - name: ICP_IDENTITY_MANAGER_IMAGE
```
Comment (posriniv) [inline deploy/olm-catalog/ibm-iam-operator/3.9.1/ibm-iam-operator.v3.9.1.clusterserviceversion.yaml:1716]: @rashmi43 you need to change the image version here as well for identity provider and manager

**Review by posriniv — CHANGES_REQUESTED** [review]

File: deploy/olm-catalog/ibm-iam-operator/3.9.1/ibm-iam-operator.v3.9.1.clusterserviceversion.yaml (hunk 1 above)
Comment (rashmi43) [reply deploy/olm-catalog/ibm-iam-operator/3.9.1/ibm-iam-operator.v3.9.1.clusterserviceversion.yaml:1716]: done

**Review by posriniv — APPROVED** [review]
lgtm

**Review by pnpavan25 — APPROVED** [review]
lgtm

**Pull request merged**